docs/DeveloperGuide.adoc

```diff
@@ -838,21 +838,217 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
-
-|`* * *` |user |add a new loan |
-
-|`* * *` |user |delete a loan |remove entries that I no longer need
-
-|`* * *` |user |find a loan by name |locate details of loans without having to go through the entire list
-
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
-
-|`*` |user with many loans in the address book |sort loans by name |locate a loan easily
+|`* * *`
+|bicycle rental shop owner who wants to manage my bicycles
+|register my bicycles into the app
+|keep track of my bicycles within the app
+
+|`* * *`
+|bicycle rental shop owner who wants to manage my bicycles
+|edit details for my bicycles within the app
+|update the system with the latest bicycle statuses (e.g. got damaged, repaired, changed name or ID)
+
+|`* * *`
+|bicycle rental shop owner who wants to manage my bicycles
+|remove bicycles from the app
+|reflect the decommissioning of bicycles in the app
+
+|`* *`
+|bicycle rental shop owner who wants to manage my bicycles
+|track the features of my bicycles (e.g. wheel size, gears, has a bell)
+|search for particular bicycles or types of bicycles better, especially when serving customers
+
+|`*`
+|bicycle rental shop owner who wants to manage my bicycles
+|mark a bicycle with a repair end date
+|I can be reminded by the app on when to collect my bike from the repairman
+
+|`* * *`
+|bicycle rental shop owner who wants to manage my loans
+|initiate new loans into the app
+|keep track of my loans within the app
+
+|`* *`
+|bicycle rental shop owner who wants to manage my loans
+|edit a loan's details
+|correct a typo or adapt to a customer's change of mind
+
+|`* * *`
+|bicycle rental shop owner who wants to manage my loans
+|register the completion of a loan (i.e. return of the bike) in the system
+|update the system (make the bike available again, receive payment etc.)
+
+|`* * *`
+|bicycle rental shop owner who wants to manage my loans
+|cancel a loan
+|update the system in case the loan is invalidated (e.g. the customer changes their mind)
+
+|`* * *`
```

Comment:
Repeat of the previous one. Please delete this. Thanks.

Reply:
It's not a repeat.
The first one is to register a loan as complete, the bicycle is returned, payment is made, the loan is archived etc.
The second one is to cancel a loan, as if it never happened.